cmd/bulkcopy.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type BcpArguments struct {
+	SchemaObjectName     string
+	QueryText            string
+	BatchSize            int
+	ErrorFile            string
+	FormatFile           string
+	FirstRow             int64
+	Hints                string
+	InputFile            string
+	LastRow              int64
+	MaxErrors            int
```

Comment:
FYI i'm redoing this struct and putting it in /pkg/bulkcopy

Reply:
Sounds good